config/src/lib.rs

```diff
@@ -60,6 +61,20 @@ pub enum PrivacyType {
 	Private,
 }
 
+impl PrivacyType {
+	fn to_object<'a, C: Context<'a>>(&self, cx: &mut C) -> JsResult<'a, JsObject> {
```

Comment:
Is is possible to define  a trait to convert to JsObject and only implement it inside  `bridge/node` instead of adding it here?

Reply:
Ideally not having neon dependency on this crate if possible

Reply:
Yep that's a goal, was just dipping my toes in neon fluid